ci/check-crates.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# env:
+#   - CRATE_TOKEN
+
+declare skip_patterns=(
+    "./Cargo.toml"
+    "./programs/sbf/Cargo.toml"
+)
+
+declare -A verified_crate_owners=(
+    ["solana-grimes"]=1
+)
+
+# search all Cargo.toml
+readarray -t files <<<"$(find . -name "Cargo.toml")"
```

Comment:
I search the whole repos because I would like to including new added crates's Cargo.toml automatically.
If we make this set manually, I think we will miss some new crates checking if someone don't add the Cargo.toml.🤔